src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int numberRandom = new Random().nextInt(100);
```

Comment:
```suggestion
    private final Random random = new Random();
```
you need new number every time